<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectDir</class>
 <widget class="QDialog" name="selectDir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>115</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>300</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>文件打包工具</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/packageIcon</normaloff>:/icon/packageIcon</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="Icon">
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="text">
        <string>继续</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rejectButton">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="appName">
         <property name="text">
          <string>文件打包工具</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ver">
         <property name="text">
          <string>v0.0.1(dev in Qt)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="dirMode">
       <item>
        <widget class="QLabel" name="dirIndicator">
         <property name="text">
          <string>请选择项目文件夹所在的目录:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="currentPath">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>程序当前目录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="usrPath">
         <property name="text">
          <string>另行选择</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="dirBox">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="pathEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="dirButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rejectButton</sender>
   <signal>clicked()</signal>
   <receiver>selectDir</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
